build/Dockerfile

```diff
@@ -1,26 +1,47 @@
-FROM centos:centos8
-
+FROM quay.io/centos/centos:stream8
+# FROM registry.access.redhat.com/ubi8/ubi:8.1
 # Use this build arg to set any default test script arguments
-ARG RUN_SCRIPT_ARGS=
 ENV RUN_SCRIPT_ARGS=${RUN_SCRIPT_ARGS}
-
+ENV ROBOT_EXTRA_ARGS=${ROBOT_EXTRA_ARGS}
 ENV HOME /tmp
 
+ARG oc_local_path=build/oc
+
```

Comment:
Do you think it will be good idea to upload oc file to git repo(personal/ods-ci) and use ADD to directly download and move it to  /usr/local/bin/oc?

Reply:
not sure. If we do that, we should also maintain it and update it. However, let's wait until we understand if this is something that should be implemented directly in openshiftcli library

Reply:
starting from @jgarciao 's suggestion, I've implemented the download of oc  client  directly from `https://mirror.openshift.com/pub/`. The version and channel (e.g., candidate, stable, etc) to be downloaded can be selected by using OC_VERSION and OC_CHANNEL arguments.